frameworks/Go/goji/bash_profile.sh

```diff
@@ -0,0 +1,4 @@
+# Set the root of our go installation
+export GOROOT=${IROOT}/go
+
+export GOPATH=${TROOT}
```

Comment:
Can you move these variables to your setup.sh and/or install.sh and then remove the bash_profile.sh file. We just merged a change to get rid of the need for this file entirely. (issue #1478)